unafamilia/src/main/java/com/unfamilia/eggbot/domain/raidpackage/Item.java

```diff
@@ -15,6 +15,7 @@
 @NoArgsConstructor
 @EqualsAndHashCode(callSuper = false)
 @Table(name = "item", indexes = {
+        @Index(name = "item_id", columnList = "item_id"),
```

Comment:
Primary key is a special kind of index and you don't need to define the `@Index` annotation onto it, as `@Id` covers that